cms/static/js/views/list.js

```diff
@@ -36,11 +36,17 @@ define([
         },
 
         render: function(model) {
+            // Cannot create more then one object/entity if model 'singleEntity' marked as true.
+            var singleEntity = false;
```

Comment:
See above comment regarding the change of this parameter to a 'maxAllowed' variable which contains the number of supported certificates